pmm-app/src/shared/components/Form/NumericInput/NumericInput.style.ts

```diff
@@ -0,0 +1,100 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { stylesFactory } from '@grafana/ui';
+
+export const getStyles = stylesFactory((theme: GrafanaTheme) => {
+  const arrow = css`
```

Comment:
For the speed sake it's better to put constants which do not depend on anything outside of the function.

Reply:
Right, moving `arrow` and `baseInput` outside